scripts/solve_network.py

```diff
@@ -275,12 +282,13 @@ def solve_network(n, config, opts='', **kwargs):
     if 'snakemake' not in globals():
         from helper import mock_snakemake
         snakemake = mock_snakemake(
-            'solve_network',
+            'solve_network_myopic',
```

Comment:
was there a reason to rename?

Reply:
no, it was just for testing. We can change it back to the previous `solve_network`